lesson01.calculator/index.html

```diff
@@ -1,22 +1,294 @@
+                    </tr>
+                </thead>
+                <tbody>
+                    <tr>
+                        <td><input type="button" id="num7" value="7" onclick="input(this.value)"></td>
+                        <td><input type="button" id="num8" value="8" onclick="input(this.value)"></td>
+                        <td><input type="button" id="num9" value="9" onclick="input(this.value)"></td>
+                        <td><input type="button" id="opDiv" value=" / " onclick="input(this.value)"></td>
+                        <td rowspan="2"><input type="button" id="opClr" value="C" onclick="clearCalc()"></td>
+                    </tr>
+                    <tr>
+                        <td><input type="button" id="num4" value="4" onclick="input(this.value)"></td>
+                        <td><input type="button" id="num5" value="5" onclick="input(this.value)"></td>
+                        <td><input type="button" id="num6" value="6" onclick="input(this.value)"></td>
+                        <td><input type="button" id="opMul" value=" * " onclick="input(this.value)"></td>
+                    </tr>
+                    <tr>
+                        <td><input type="button" id="num1" value="1" onclick="input(this.value)"></td>
+                        <td><input type="button" id="num2" value="2" onclick="input(this.value)"></td>
+                        <td><input type="button" id="num3" value="3" onclick="input(this.value)"></td>
+                        <td><input type="button" id="opMin" value=" - " onclick="input(this.value)"></td>
+                        <td rowspan="2"><input type="button" id="opEql" value="=" onclick="calculator()"></td>
+                    </tr>
+                    <tr>
+                        <td colspan="2"><input type="button" id="num0" value="0" onclick="input(this.value)"></td>
+                        <td><input type="button" id="opDot" value="." onclick="input(this.value)"></td>
+                        <td><input type="button" id="opPls" value=" + " onclick="input(this.value)"></td>
+                    </tr>
+                </tbody>
+            </table>
+        </div>
+
     </main>
+    <script>
+
+        let eqvPrs = true;
+        let dotFlag = false;
+
+        function clearCalc() {
+            document.getElementById("calcInput").innerHTML = 0;
+            document.getElementById("calcRe").innerHTML = null;
+            eqvPrs = true;
+            dotFlag = false;
+        }
+
+        function input(value) {
+            let onScreen = document.getElementById("calcInput").innerHTML;
+            let onScreenLest = onScreen[onScreen.length - 2];
+
+            switch (onScreenLest) {
+
+                case "*":
+
+                    dotFlag = false;
+
+                    if (value === " + " || value === " - " || value === " / ") {
+                        document.getElementById("calcInput").innerHTML = onScreen.slice(0, onScreen.length - 3);
```

Comment:
You get the data from the display, calculate and show the result altogether. 
try to save the data to variables and reflect it on the display